source/compatibility.tex

```diff
@@ -151,8 +151,8 @@
   int tag;
 };
 
-static_assert(!allocator_traits<MyAlloc<int>>::is_always_equal);        // Error in \CppXXIII{},
-                                                                        // OK in \CppXXVI{}
+static_assert(!allocator_traits<MyAlloc<int>>::is_always_equal);    // Error in \CppXXIII{},
+                                                                    // OK in \CppXXVI{}
```

Comment:
CppCCVI still expands to "C++ 2026". Will they complain that it should be "this document"?

Reply:
Good point. Comparisons to the status quo are going to be less clear like tihs.

I'm inclined to not put this change in the WD. We can say "this document" in the IS.

Reply:
This entire hunk doesn't apply to C++23, the document currently with ISO/CS.